zproject/prod_settings_template.py

```diff
@@ -120,12 +120,43 @@
     # 'zproject.backends.GoogleAuthBackend',  # Google auth, setup below
     # 'zproject.backends.GitHubAuthBackend',  # GitHub auth, setup below
     # 'zproject.backends.GitLabAuthBackend',  # GitLab auth, setup below
+    # 'zproject.backends.AppleAuthBackend',  # Apple auth, setup below
     # 'zproject.backends.AzureADAuthBackend',  # Microsoft Azure Active Directory auth, setup below
     # 'zproject.backends.SAMLAuthBackend', # SAML, setup below
     # 'zproject.backends.ZulipLDAPAuthBackend',  # LDAP, setup below
     # 'zproject.backends.ZulipRemoteUserBackend',  # Local SSO, setup docs on readthedocs
 )  # type: Tuple[str, ...]
 
+########
+# Apple OAuth.
+#
+# To set up Sign In with Apple, you'll need to do the following:
+#
+# (1) Visit https://developer.apple.com/account/resources/,
+# Follow the instructions in
+# https://help.apple.com/developer-account/?lang=en#/dev77c875b7e
+# to create a Sign In Certificate.
+#
+# (2) Follow instructions in
+# https://help.apple.com/developer-account/?lang=en#/dev1c0e25352.
+# to Create a Services ID. In services ID, fill the redirect URI with
+# a value like `http://zulip.example.com/complete/apple-id` .
+#
+# (3) You'll now have a client_id(eg: com.application.your), aka "Service ID",
+# a Key ID and a Private key file. Use the client_id as SOCIAL_AUTH_APPLE_ID_CLIENT
```

Comment:
And is there something in particular that the name "client_id" refers to, or can we just say "Services ID" here again?

Reply:
we can just say "service ID" but the `python-social-auth` has it named `SOCIAL_AUTH_APPLE_ID_CLIENT`. so thought using `client_id` is good.
I referred the doc in this [PR](https://github.com/python-social-auth/social-docs/pull/68/files#diff-956fdef252a7f17b6e232fe3c182ce05) for writing those instructions, which is why I used the name `client_id`. I think changing it and having just "service ID" is fine as we'll have the instruction clearly to set that `SOCIAL_AUTH_APPLE_ID_CLIENT`.

Reply:
Sounds good. Note though that it's "Services ID" -- Apple's docs consistently spell and capitalize it that way, and to avoid confusion we should do the same.